docs/source/slurm.rst

```diff
@@ -92,20 +90,111 @@ To train a model using multiple-nodes do the following:
 
         sbatch submit.sh
 
-.. note:: using :class:`~torch.utils.data.distributed.DistributedSampler` is already handled by Lightning.
+.. note::
+    When running in DDP mode, any errors in your code will show up as an NCCL issue.
+    Set the `NCCL_DEBUG=INFO` flag to see the ACTUAL error.
+
+
+Normally now you would need to add a
+:class:`~torch.utils.data.distributed.DistributedSampler` to your dataset, however
+Lightning automates this for you. But if you still need to set a sampler set the Trainer flag
+:paramref:`~pytorch_lightning.Trainer.replace_sampler_ddp` to ``False``.
+
+Here's an example of how to add your own sampler (again no need with Lightning).
```

Comment:
```suggestion
Here's an example of how to add your own sampler (again, not need with Lightning).
```

Reply:
Thanks for proof reading @lgvaz 
This one here becomes incorrect english. 
We could change it to "not needed ..."

Reply:
Perfect! Good catch =)